GCS/probe0/probe0.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+import "github.com/google/cloudprober/probes/proto/config.proto";
+import "github.com/googleinterns/step224-2020/config/proto/probe.proto";
+option go_package = "github.com/googleinterns/step224-2020/GCS/probe0/proto";
+
+//The probe0 package is the package for the test probe - creates files in GCS with contents without any significance
```

Comment:
Rephrased this a little. Also, you may want to consider either just using "hermes" for the package name since sending these probes is one of Hermes's core duties.

```suggestion
// The probe0 encapsulates the test probe, which creates files in GCS with randomized contents.
```

Reply:
Or maybe you could follow cloudprober's package nomenclature and call this package: hermes.probes. ¯\_(ツ)_/¯

Reply:
Nit: add space after //.